src/Verbs/InitCommand.cs

```diff
@@ -0,0 +1,259 @@
+            node.Add(new XElement("Templates"));
+            node.Add(new XElement("Stylesheets"));
+            node.Add(new XElement("Macros"));
+            node.Add(new XElement("DictionaryItems"));
+            node.Add(new XElement("Languages"));
+            node.Add(new XElement("DataTypes"));
+
+            return node;
+
+        }
+
+        /// <summary>
+        ///  Workout the URL for the licence based on the string value
+        /// </summary>
+        /// <param name="licenceName">Licence Name (e.g MIT)</param>
+        /// <returns>URL for the licence file</returns>
+        private static string GetLicenceUrl(string licenceName)
+        {
+            // TODO - get licence urls from somewhere?
+            if (licenceName.Equals("MIT", StringComparison.InvariantCultureIgnoreCase))
+            {
+                return "https://opensource.org/licenses/MIT";
+            }
+
+            return string.Empty;
+        }
+        
+        /// <summary>
+        ///  Prompts the user for version string and validates it.
+        /// </summary>
+        /// <param name="prompt">text to put in prompt</param>
+        /// <param name="defaultValue">default value if user just presses enter</param>
+        /// <returns>SemVersion compatable version</returns>
+        private static SemVersion GetVersionString(string prompt, string defaultValue)
+        {
+            while (true)
+            {
+                var versionString = GetUserInput(prompt, defaultValue);
+                if (SemVersion.TryParse(versionString, out SemVersion version))
+                {
+                    return version;
+                }
+                else
+                {
+                    Console.WriteLine(Resources.Init_InvalidVersion, versionString);
+                }
+            }
+        }
+
+        /// <summary>
+        ///  Prompt the user for some input, return a default value if they just press enter
+        /// </summary>
+        /// <param name="prompt">Prompt for user</param>
+        /// <param name="defaultValue">Default value if they just press enter</param>
+        /// <returns>user value or default</returns>
+        private static string GetUserInput(string prompt, string defaultValue)
+        {
+            Console.Write($"{prompt}: ");
+            if (!string.IsNullOrWhiteSpace(defaultValue))
+            {
+                Console.Write($"({defaultValue}) ");
+            }
+
+            var value = Console.ReadLine();
+            if (string.IsNullOrWhiteSpace(value))
+                return defaultValue;
+
+            return value;
+        }
+
+        /// <summary>
+        ///  Validates the path to where the package file is going to be created
+        /// </summary>
+        /// <param name="packageFile">path to a package file</param>
+        private static string GetPackageFile(string packageFile)
+        {
+            var currentFolder = Path.GetDirectoryName(AppContext.BaseDirectory);
```

Comment:
`currentFolder` is never used it seems, can we remove? 🙂

Reply:
removed